Library/Formula/ccat.rb

```diff
@@ -0,0 +1,14 @@
+require "formula"
+
+class Ccat < Formula
+  homepage "https://github.com/jingweno/ccat"
+  url "https://github.com/jingweno/ccat/archive/v0.1.0.tar.gz"
+  sha1 "fc0a1a49ca81fc85b9d50f48d63182d507b4627c"
```

Comment:
Yes. `sha256` is required for all new formulae, all existing `sha1` bottles are being migrated to `sha256` as they are updated and there's a strong ongoing effort to eventually deprecate `sha1` from formulae entirely.